docs/user/accumulo/commandline_tools.rst

```diff
@@ -168,12 +168,12 @@ If there is whitespace within a keyword, enclose it in quotes for proper functio
       -i instance -z zoo1,zoo2,zoo3 \
       -c catalog -f featureTypeName
 
-get-names
-~~~~~~~~~
+get-type-names
```

Comment:
Did all the instances of 'get-names' get updated?

Reply:
i think so... if not here then in the docs pr. might have missed some of course...